packit/cli/generate.py

```diff
@@ -36,7 +36,7 @@
 from packit.constants import CONFIG_FILE_NAMES, PACKIT_CONFIG_TEMPLATE
 from packit.exceptions import PackitException
 
-logger = logging.getLogger(__file__)
+logger = logging.getLogger("packit")
```

Comment:
In other places, we are using `__name__`:

```suggestion
logger = logging.getLogger(__name__)
```

(But ignore me if this is right -- I do not understand python-logging a lot.)

Reply:
I see, but what you say is unfortunately not a rule for cli:
```
[dhodovsk@shiny packit]$ grep -r 'Logger' packit/cli/
packit/cli/packit_base.py:logger = logging.getLogger("packit")
packit/cli/types.py:logger = logging.getLogger(__name__)
packit/cli/status.py:logger = logging.getLogger(__file__)
packit/cli/create_update.py:logger = logging.getLogger(__file__)
packit/cli/push_updates.py:logger = logging.getLogger(__file__)
packit/cli/utils.py:logger = logging.getLogger(__name__)
packit/cli/create_sg.py:logger = logging.getLogger(__file__)
packit/cli/sync_from_downstream.py:logger = logging.getLogger(__file__)
packit/cli/update.py:logger = logging.getLogger(__file__)
packit/cli/generate.py:logger = logging.getLogger("packit")
packit/cli/srpm.py:logger = logging.getLogger("packit")
packit/cli/build.py:logger = logging.getLogger(__file__)
```